givitsite/templates/coordinator.html

```diff
@@ -0,0 +1,144 @@
+            <option value="Haifa">Haifa</option>
+            <option value="Rosh Haain">Rosh Haain</option>
+            <option value="Modiin">Modiin</option>
+            <option value="Beer Sheva">Beer Sheva</option>
+        </select>
+    </div>
+
+    <form method='POST' action="/coordinate/create_brief" style="margin:auto; padding:3px; width:30%"> {%csrf_token%}
+        <div class="row" style="display:inline">
+            <input type="date" class="form-control mb-2" name="date" id="inlineFormInput" placeholder="choose date" style="width:50%">
+            <button type="submit" class="btn btn-primary mb-2" style="width:50%">create brief</button>
+        </div>
+    </form>
+
+
+{% for item in matchedItems %}
+    {% for request in openRequests %}  
+       {% if item.request_id == request.id %}
+                <div>
+                    <ul>
+                        <table class="table table-bordered table-striped" style="width:85%; margin-top:20px">
+                            <tr>
+                                <td>Item type</td>
+                                <td>pick up</td>
+                                <td>drop off</td>
+                                <td>URL</td>
+                                <td>Pic</td>
+                                <td>coordinate form</td> 
+                            </tr>
+                            <tr>
+                                <td>{{item.title}}</td>
+                                <td>{{item.city}}</td>
+                                <td>{{item.city}}</td>
+                                <td><a href="{{item.url}}">link</a></td>
+                                <td><img src="{{item.picture}}" style='width:150px; height:150px'/></td>
+                                <td><button type='input' class="btn btn-primary"     
+                                onclick="set_for_coordination('{{item.title}}','{{request.friend_id}}', '{{item.request_id}}', '{{item.city}}', '{{item.city}}','{{item.picture}}');">
+                                open</button>
+                                <button type='input' class="btn btn-danger"     
+                                        style="margin-left:10px" onclick="hide_form();">close</button>
+                                </td>
+                            </tr>
+                        </table>
+                    </ul>
+                </div>
+
+                <div class="jumbotron center" id="coordination-div" hidden="true" style="margin-bottom:10%; margin:auto; width:60%; border:3px solid white; padding:10px; background:lavender">
+                    <h1 class="display-4" style="text-align:center">COORDINATION FORM</h1> 
+                    <h5 style="text-align:center">Insert Dates and times for transfer after coordinating with the following contacts:</h5> 
+                    <form  method="POST" hidden="true" id="coordination-form" style="margin: auto; width:60%; margin-top:7%">    {%csrf_token%}
```

Comment:
Django can probably auto-generate the forms for you.

Reply:
now it does:)